keyboards/lily58/keymaps/paulomp90/config.h

```diff
@@ -0,0 +1,41 @@
+/* Copyright 2023 Paulo Pereira
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define MASTER_LEFT
+
+#define QUICK_TAP_TERM 0
+#define TAPPING_TERM 150
+
+#undef RGBLED_NUM
+#define RGBLIGHT_EFFECT_BREATHING
+#define RGBLIGHT_EFFECT_RAINBOW_MOOD
+#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
+#define RGBLIGHT_EFFECT_SNAKE
+#define RGBLIGHT_EFFECT_KNIGHT
+#define RGBLIGHT_EFFECT_CHRISTMAS
+#define RGBLIGHT_EFFECT_STATIC_GRADIENT
+#define RGBLIGHT_EFFECT_RGB_TEST
+#define RGBLIGHT_EFFECT_ALTERNATING
+#define RGBLIGHT_EFFECT_TWINKLE
+#define RGBLED_NUM 27
```

Comment:
This is for the r2g board?

Reply:
Yes, but I have some doubts about these `rgblight` options.

Are they unnecessary?

Reply:
They shouldn't be necessary.  And by default, the r2g uses RGB matrix (per key) for lighting, not RGB Light (underglow).  

And in general, these settings are wrong for how the board's hardware is configured, anyways.  So you could safely remove the entire block of RGB Light settings.

Reply:
@drashna removed, thank you very much 🙇‍♂️